validate-remote-docker-config pipeline not working

I tried queuing a build to validate an agent that uses a remote Docker daemon.  The agent is known to be operating correctly.  The build failed in the `Pull Ping Image` step with this error:

```
Generating script.
Script contents:
docker run --rm --entrypoint docker -v /var/run/docker.sock:/var/run/docker.sock -v /data/docker-certs/ddvsotx2l043:/docker-certs -e DOCKER_CERT_PATH=/docker-certs -e DOCKER_TLS_VERIFY=1 -e DOCKER_HOST=tcp://10.197.175.56:2376 mcr.microsoft.com/dotnet-buildtools/prereqs:debian-stretch-docker-testrunner-63f2145-20184325094343 pull
========================== Starting Command Output ===========================
[command]/bin/bash --noprofile --norc /data/agenta/_work/_temp/4438f5da-0c37-41c3-a11c-44dbaa85ff64.sh
"docker pull" requires exactly 1 argument.
See 'docker pull --help'.

Usage:  docker pull [OPTIONS] NAME[:TAG|@DIGEST] [flags]

Pull an image or a repository from a registry
##[error]Bash exited with code '1'.
##[section]Finishing: Pull Ping Image
```

--- a/eng/pipelines/templates/variables/common.yml
+++ b/eng/pipelines/templates/variables/common.yml
@@ -2,3 +2,7 @@ variables:
 - template: ../../../common/templates/variables/common.yml
 - name: checkBaseImageSubscriptionsPath
   value: eng/check-base-image-subscriptions.json
+- name: imageNames.pinger.linux
+  value: mcr.microsoft.com/windows/nanoserver:1809-arm32v7
+- name: imageNames.pinger.windows
+  value: arm32v7/debian:stretch

